lint/rule.go

```diff
@@ -240,24 +240,46 @@ func (r *EnumRule) Lint(fd *desc.FileDescriptor) []Problem {
 // ruleIsEnabled returns true if the rule is enabled (not disabled by the comments
 // for the given descriptor or its file), false otherwise.
 func ruleIsEnabled(rule ProtoRule, d desc.Descriptor) bool {
-	directive := fmt.Sprintf("api-linter: %s=disabled", rule.GetName())
+	name := string(rule.GetName())
+	if ruleDisabledInComments(name, d) {
+		return false
+	}
+
+	// Some rules have a legacy alias.
+	if alias := ruleAlias(name); alias != "" && ruleDisabledInComments(alias, d) {
+		return false
+	}
+
+	return true
+}
+
+// ruleAlias returns the rule alias. If not exists, return empty string.
+func ruleAlias(name string) string {
+	aliasMap := map[string]string{
+		"core::0140::lower-snake": "naming-format",
+	}
+	return aliasMap[name]
+}
 
-	// If the comments above the descriptor disable the rule,
-	// return false.
+// ruleDisabledInComments returns true if the rule is disabled by the comments
+// for the given descriptor or its file.
+func ruleDisabledInComments(ruleName string, d desc.Descriptor) bool {
```

Comment:
Instead of a separate function, just make a slice of strings `[]directive`, with the rule name and the alias, and `range` over it. It will be much simpler.

Reply:
Done. Thanks!